lerna.json

```diff
@@ -1,12 +1,4 @@
 {
   "packages": ["packages/*", "examples/*", "e2e/project", "e2e/packages/*"],
-  "version": "independent",
-  "command": {
-    "publish": {
-      "allowBranch": "master",
-      "conventionalCommits": true,
-      "githubRelease": true,
-      "message": "chore(release): publish [skip ci]"
-    }
-  }
```

Comment:
We are using `changesets` instead of `lerna` to publish our packages so that is not necessary anymore.

Sorry to have done that here but something failed in lerna, I don't remember exactly what, so I removed that code to fix it and because we don't use it anymore.

Reply:
oh I see, thanks :)